### PR TITLE
Added users controller, fixed some route paths

### PR DESCRIPTION
**Pull request by herunan**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    redirect_to root_path
+  end
+
+  private
+
+  def user_params
+    params.require(:user).permit(:first_name, :last_name, :email, :phone_number)
+  end
+
+  def set_cocktail
```
Comment (baldipa3) [inline app/controllers/users_controller.rb]: Change method name to set_user
`def set_user`
`@user = User.find(params[:id])`
`end`

File: config/routes.rb [hunk 2]
```diff
@@ -1,10 +1,10 @@
 Rails.application.routes.draw do
-  devise_for :users
+  devise_for :users, except: %i[index]
   root to: 'pets#index'
   resources :pets do
     resources :bookings
   end
   resources :bookings do
-    resources :reviews, only: [:new, :create, :edit, :destroy]
```
Comment (baldipa3) [inline config/routes.rb:8]: How are we going to create reviews?

**Review by baldipa3 — CHANGES_REQUESTED** [review]
Review done

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (nosuchcoder) [reply app/controllers/users_controller.rb]: Why do we need a set method inside the user controller, isn't that only needed inside reviews

**Review by nosuchcoder — CHANGES_REQUESTED** [review]
Remove the def_set cocktail at the end of the user controller

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (herunan) [reply app/controllers/users_controller.rb]: It helps refactor the code against repetition of the .find method

File: config/routes.rb (hunk 2 above)
Comment (herunan) [reply config/routes.rb:8]: Probably thought it was `except:`

**Pull request merged**